tests/components/matter/snapshots/test_sensor.ambr

```diff
@@ -3412,6 +3412,72 @@
     'state': '28.3',
   })
 # ---
+# name: test_sensors[vacuum_cleaner][sensor.mock_vacuum_operational_state-entry]
+  EntityRegistryEntrySnapshot({
+    'aliases': set({
+    }),
+    'area_id': None,
+    'capabilities': dict({
+      'options': list([
+        'stopped',
+        'running',
+        'paused',
+        'error',
+        'seeking_charger',
+        'charging',
+        'docked',
+      ]),
+    }),
```

Comment:
doesn't a vacuum entity already represent these modes?

Reply:
> doesn't a vacuum entity already represent these modes?

some of it, not all. 
Also OperationalState is optional so we offer it as optional sensor while keeping the vacuum on RvcCleanMode cluster ( I need to adjust that in another PR).

This at least makes it consistent across all devices

Reply:
More complete answer is also that Operation State gives a more detailed state, including vendor specific messages. 
So the vacuum entity will use the HA standardized states but Operation state sensor give smore detail and this will be uniform for all devices that are based on Operational State